### PR TITLE
Add New Idea

### LINKED ISSUE #266: ✨ [Feature request]: Add New Idea Portfolio
**Is your feature request related to a problem? Please describe.**
Nothing

**Describe the solution you'd like**
A Full Stack Portfolio made with react and sanity

**Describe alternatives you've considered**

**Additional context**
I am happy to work on

### PR DESCRIPTION
**Pull request by Programming-School-Pro-Coding**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Added A New Idea and Edit another idea made with me

Fixes #266 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Programming-School-Pro-Coding** [comment]
@atapas ...

**Comment by atapas** [comment]
@Programming-School-Pro-Coding which issue is it related to?

**Review by atapas — CHANGES_REQUESTED** [review]
Edit the PR DESCRIOPTION WITH ISSUE NUMBER

**Comment by Programming-School-Pro-Coding** [comment]
> @Programming-School-Pro-Coding which issue is it related to?

Ok

**Comment by Programming-School-Pro-Coding** [comment]
> Edit the PR DESCRIOPTION WITH ISSUE NUMBER

Done

File: src/common/playideas/ideas.json [hunk 1]
```diff
@@ -123,7 +123,13 @@
     {
       "id": "idea-21",
       "title": "Animations Portfolio",
-      "description": "Build a Portfolio made with react and sanity by framer-motion. Projects, Portfolio Section, Skills & Experience, Companies you work in with carousel, and Contacts.",
+      "description": "Build a Portfolio made with react by framer-motion. Projects, Portfolio Section, Skills & Experience, Companies you work in with carousel, and Contacts.",
```
Comment (atapas) [inline src/common/playideas/ideas.json:126]: What's the difference between these two projects? Looks almost similar to me.

Comment (atapas) [reply src/common/playideas/ideas.json:126]: Can you make it a little more clear. The difference in the descriptions are not very clear. It looks almost similar to me(and someone else told the same)

Comment (Programming-School-Pro-Coding) [reply src/common/playideas/ideas.json:126]: > Can you make it a little more clear. The difference in the descriptions are not very clear. It looks almost similar to me(and someone else told the same)

Ok

**Review by atapas — APPROVED** [review]

**Pull request merged**